pkg/pulumiyaml/run.go

```diff
@@ -60,6 +60,11 @@ func LoadFromCompiler(compiler string, workingDirectory string) (*ast.TemplateDe
 	cmd.Dir = workingDirectory
 	cmd.Stdout = &stdout
 	cmd.Stderr = &stderr
+	cmd.Env = append(os.Environ(), env...)
```

Comment:
```suggestion
	cmd.Env = append(env, os.Environ()...)
```

I think I would still be more comfortable doing it this way around (and adjusting the warning message).  That would make this change 100% backwards compatible, otherwise I'm a bit worried about that.

Reply:
Fine by me 👍